### ISSUE TITLE
A11y_AzureDataStudioAAE_Azure ADC(Preview)_Azure Configuration_Keyboard: Keyboard focus is not visible on "Next" button while navigating with help of tab key.

### ISSUE THREAD

**Issue by sspatil123** [microsoft/azuredatastudio#15715]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yMAS;#A11yTCS;#A11ySev2;#SH_AzureDataStudio_AAE_Jun2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#Keyboard;#WCAG2.4.7;#AINotInScope;

### Environment Details:
Application Name: Azure Data Studio-AAE
Windows Version: Win10

### Repro Steps:

1. Launch the application-Azure Data Studio-Insiders and install extension Azure arc and Azure data ClI
2. Connect to Azure Arc Controller
3. Tab till "Connection" and tab to see more button
4. Tab to "New Deployment" and Azure ADC(Preview) and then tab select button
5. Deployment Prerequisite page will open and tab on next "Select from existing Kubernetes cluster" will open
6. Tab till  "Next" button and see focus

### Actual:
Keyboard focus is not visible on "Next" button as there is no outline is visible on next button while navigating with help of tab key.

Similar issue is present throughout the application where buttons have blue background.

### Expected:
Keyboard focus should be visible on  "Next" button while navigating with help of tab key, Change to Keyboard focus should be properly visible as outline..

### User Impact:
Low vision users will not know where the keyboard focus goes on "Next" button or not.

**Comment by Charles-Gagnon** [comment]
This has been fixed with the latest VS Code merge

![image](https://user-images.githubusercontent.com/28519865/126844792-f94658bb-26f2-4864-b199-a80c2ce6f34e.png)

**Comment by Shivamd1** [comment]
closing not repro
![image](https://user-images.githubusercontent.com/60880975/150973803-d5fae333-89db-4c00-8941-61b344e9a316.png)

**Comment by Harika1702** [comment]
GitHub Tags: #Closed;
